src/styles/blocks/catalog.scss

```diff
@@ -18,8 +20,16 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: 300ms;
```

Comment:
It would be great to create a variable for transition duration as it should be the same for whole page